framework/doc/content/syntax/Outputs/index.md

```diff
@@ -168,6 +168,7 @@ section near the top of the simulation that shows the settings being used for ea
 | "timestep_begin" | executes the output method at the beginning of the timestep |
 | "final"          | calls the output method on the final timestep |
 | "failed"         | executes the output method when the solution fails |
+| "multiapp_fixed_point_iteration_end" | executes the output method at the end of each successful MultiApp fixed point iteration (only if fixed point iterations are enabled) |
```

Comment:
I guess this does not work with `Exodus` and some other output types. If so, need to document what types this execute works with and might want to throw errors for those types if this is set (or make the execute flag only available in those supported output types).

Reply:
do you need this? isnt it the same as timestep_end?

Reply:
Yes, we need this: it's explained in https://github.com/idaholab/moose/issues/31110; essentially, output on TIMESTEP_END occurs after the end of the fixed point loop (I suppose since that's what users would expect and usually want).